src/main.rs

```diff
@@ -52,6 +52,7 @@ const VERSION: &str = env!("CARGO_PKG_VERSION");
 const TOASTFILE_DEFAULT_NAME: &str = "toast.yml";
 const CONFIG_FILE_XDG_PATH: &str = "toast/toast.yml";
 const DEFAULT_LOG_LEVEL: LevelFilter = LevelFilter::Info;
+const DOCKER_CLI_DEFAULT: &str = "docker";
```

Comment:
Why is this in `main.rs` and not with `REPO_DEFAULT` in `config.rs`? As far as I can tell, it's only used in `config.rs`, and is never referenced in this file. At the very least, `DOCKER_CLI_DEFAULT` and `REPO_DEFAULT` should live in the same place, since they are the defaults for the corresponding fields in the config file schema.

Reply:
Good catch, that was a remnant of the previous implementation where it was used in two locations not just `docker.rs` I'll fix it first thing in the morning.